src/webextension/locales/en-US/manage.ftl

```diff
@@ -38,6 +38,7 @@ item-details-title = Title
 item-details-origin = Origin
 item-details-username = Username
```

Comment:
I'd really like the ability to copy the username, too.

Reply:
>I'd really like the ability to copy the username, too.

Great point 👍 I have some complex usernames and a shortcut here too would be nice.